documentation/src/main/asciidoc/topics/proc_configuring_declaratively.adoc

```diff
@@ -0,0 +1,137 @@
+[[cache_configuration_declarative]]
+= Declarative Cache Configuration
+
+Declarative configuration comes in a form of XML document that adheres to a provided {brandname} configuration XML
+link:http://www.infinispan.org/schemas/infinispan-config-{infinispanversion}.xsd[schema].
+
+Every aspect of {brandname} that can be configured declaratively can also be configured programmatically.
+In fact, declarative configuration, behind the scenes, invokes the programmatic configuration API as the XML configuration file is being processed.
+One can even use a combination of these approaches.
+For example, you can read static XML configuration files and at runtime programmatically tune that same configuration.
+Or you can use a certain static configuration defined in XML as a starting point or template for defining additional configurations in runtime.
+
+There are two main configuration abstractions in {brandname}: `global` and `cache`.
+
+.Global configuration
+Global configuration defines global settings shared among all cache instances created by a single link:{javadocroot}/org/infinispan/manager/EmbeddedCacheManager.html[EmbeddedCacheManager].
+Shared resources like thread pools, serialization/marshalling settings, transport and network settings, JMX domains are all part of global configuration.
+
+.Cache configuration
+Cache configuration is specific to the actual caching domain itself: it specifies eviction, locking, transaction, clustering, persistence etc.
+You can specify as many named cache configurations as you need. One of these caches can be indicated as the `default` cache,
+which is the cache returned by the `CacheManager.getCache()` API, whereas other named caches are retrieved via the `CacheManager.getCache(String name)` API.
+
+Whenever they are specified, named caches inherit settings from the default cache while additional behavior can be specified or overridden.
+{brandname} also provides a very flexible inheritance mechanism, where you can define a hierarchy of configuration templates,
+allowing multiple caches to share the same settings, or overriding specific parameters as necessary.
+
+NOTE: Embedded and Server configuration use different schemas, but we strive to maintain them as compatible as possible so that you
```

Comment:
No longer true. The server configuration just uses an additional namespace for the server bits.